sdk/trace/src/main/java/io/opentelemetry/sdk/trace/samplers/package-info.java

```diff
@@ -3,7 +3,31 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-/** Samplers for selecting traces that are recorded and exported. */
+/**
+ * This package contains {@link io.opentelemetry.sdk.trace.samplers.Sampler}s for selecting traces
+ * that are recorded and exported. <br>
+ *
+ * <p>Sampling is a mechanism to control the noise and overhead introduced by OpenTelemetry by
+ * reducing the number of samples of traces collected and sent to the backend. <br>
+ *
+ * <p>The following sampling strategies are provided here:
+ *
+ * <p>{@link io.opentelemetry.sdk.trace.samplers.Sampler#alwaysOff()} : This strategy will ensure
+ * that no Spans are ever sent to the export pipeline.
+ *
+ * <p>{@link io.opentelemetry.sdk.trace.samplers.Sampler#alwaysOn()} : This strategy will ensure
+ * that every Span will be sent to the export pipeline.
+ *
+ * <p>{@link io.opentelemetry.sdk.trace.samplers.Sampler#traceIdRatioBased(double)} : This strategy
+ * will sample the provided fraction of Spans, deterministically based on the TraceId of the Spans.
+ * This means that all spans from the a given trace will have the same sampling result.
+ *
+ * <p>{@code Sampler#parentBased(Sampler)} : This strategy will always use the sampled state of the
```

Comment:
```suggestion
 * <p>{@link Sampler#parentBased(Sampler)} : This strategy will always use the sampled state of the
```

Reply:
unfortunately, then you have to fully qualify it and it fails the formatting check. Any idea how to turn off the line-length checks for package-info files?

Reply:
Yeah, added a commit here to disable checkstyle LineLength for package-info.java specifically, I think it's safe since it's just documentation.